R/EDGE-Industry.R

```diff
@@ -1230,7 +1234,8 @@ calcSteel_Projections <- function(subtype = 'production',
 
 #' @rdname EDGE-Industry
 #' @export
-calcIndustry_Value_Added <- function(match.steel.historic.values = TRUE,
```

Comment:
please rename the file according to the function name `calcSteel_Projections`

Reply:
to follow naming conventions and ensure that caching functionality works, I would prefer to split the two functions into two files. Maybe one of them is a cal* function, one a tool* function?

Reply:
1. Filenames have nothing to do with caching.
2. The attentive observe would have noticed that both functions are called `calc…` and are called using `calcOutput()`.  No, neither of them is a `tool…` function.